### PR TITLE
Moshi moshi? minoru enhansumentsu desu

### PR DESCRIPTION
**Pull request by mid-kid**

### CODE REVIEW

File: home/names.asm [hunk 1]
```diff
@@ -125,7 +125,7 @@ GetPokemonName::
 	ld e, a
 	ld h, 0
 	ld l, a
-	add hl, hl ; hl = hl * 4
+	add hl, hl ; hl = hl * 2
 	add hl, hl ; hl = hl * 4
 	add hl, de ; hl = (hl*4) + de
```
Comment (Rangi42) [inline home/names.asm]: `; hl = (hl*4) + de = hl*5

File: home/names.asm [hunk 2]
```diff
@@ -125,7 +125,7 @@ GetPokemonName::
 	ld e, a
 	ld h, 0
 	ld l, a
-	add hl, hl ; hl = hl * 4
+	add hl, hl ; hl = hl * 2
 	add hl, hl ; hl = hl * 4
```
Comment (Rangi42) [inline home/names.asm]: `; hl = (hl*4) + de = hl*5`

File: home/names.asm [hunk 3]
```diff
@@ -125,7 +125,7 @@ GetPokemonName::
...
 	add hl, de ; hl = (hl*4) + de
 	add hl, hl ; hl = (5*hl) + (5*hl)
```
Comment (Rangi42) [inline home/names.asm]: `; hl = (hl*5) + (hl*5) = hl*10

File: home/names.asm (hunk 1 above)
Comment (Rangi42) [inline home/names.asm]: `; hl = (hl*4) + de = hl*5`

File: home/names.asm (hunk 3 above)
Comment (Rangi42) [reply home/names.asm]: I'm not sure these comments need to be here at all; "`; Each name is ten characters`" plus knowing what `add pp, qq` does tells you it's offsetting `hl` by `[wNamedObjectIndexBuffer] * 10`.

Comment (mid-kid) [reply home/names.asm]: Or just one comment at the top with `hl = hl * 10`. I think it's at least slightly useful to annotate this one as it'll differ between translations.

**Pull request merged**